### DISCUSSION TITLE
Which status checks should I use if it the check only shows if it fails?

### DISCUSSION THREAD

**Discussion by hongbo-miao** [Mergifyio/mergify#2698] [Q&A] [answered]
Most time I am using `check-success=xx` all time as auto merging rule.

Now I have a check "renovate/artifacts" by Renovate. It only shows up in GitHub checks if it fails.
https://github.com/renovatebot/renovate/discussions/10215
![image](https://user-images.githubusercontent.com/3375461/120049483-ad271800-c04c-11eb-96ac-be7093dde867.png)

Based on https://docs.mergify.io/conditions/#about-status-checks
Should I use `check-failure!=renovate/artifacts`? Thanks

**Comment by hongbo-miao** [comment, marked as answer]
Just found Config Editor at https://dashboard.mergify.io/ for testing. So convenient!
Confirmed `check-failure!=renovate/artifacts` is correct way for this case.

**Reply by jd** [reply]
Be careful when using such conditions as if `renovate/artifacts` never reports its status (e.g., it's broken), then the condition will be `true`.
